peachjam_search/forms.py

```diff
@@ -13,3 +13,9 @@ class SavedSearchUpdateForm(forms.ModelForm):
     class Meta:
         model = SavedSearch
         fields = ["note"]
+
+
+class SearchFeedbackCreateForm(forms.ModelForm):
+    class Meta:
+        model = SearchFeedback
+        fields = ["name", "email", "search_trace", "user", "feedback"]
```

Comment:
Don't allow user, otherwise I can impersonate a user. Instead, set it directly in the view based on the authenticated user. See `SavedDocumentCreateView`